cms/standard_pages/models.py

```diff
@@ -0,0 +1,39 @@
+from typing import ClassVar
+
+from django.db import models
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+from cms.core.blocks.stream_blocks import CoreStoryBlock
+from cms.core.fields import StreamField
+from cms.core.models import BasePage
+
+
+class InformationPage(BasePage):  # type: ignore[django-manager-missing]
+    """A generic information page model."""
+
+    template = "templates/pages/information_page.html"
+
+    parent_page_types: ClassVar[list[str]] = [
+        # Ensures that the information page can only be created under the home page
+        "home.HomePage",
+        # Ensures that the information page can be created under another information page
+        "standard_pages.InformationPage",
+    ]
+
+    summary = models.TextField(max_length=255)
+    last_updated = models.DateField(blank=True, null=True)
+    content = StreamField(CoreStoryBlock())
```

Comment:
Core storyblock contains equations and embeds, which requires some JS to be loaded which currently are not. See Dan's PR for details: https://github.com/ONSdigital/dis-wagtail/pull/20/files#diff-7c773641945aa15a2cb8c88a31f57b28667ff092ceaefc551f02b50929a3763fR142-R150

Reply:
49f9383 may be of help

Reply:
I looked at your PR Dan specifically around the equations and ons_embed. The latest code written in `base.py`, `has_equations ` and `has_ons_embed . I included this locally for this task and also the partial and it all works for me =)

Neha and I tested it.

But I think I'll wait till your PR gets approved and merged first and then I'll pull your code in and use it.

I was reviewing your analysis PR with Neha, and I got some questions, which I'll ask tomorrow on a call =)

Reply:
@sanjeevz3009 @nehakerung #20 is now merged, so this is ready to progress

Reply:
Merged and added 🎉

Reply:
Good job Neha! Added some comments below similar to Dan!

Reply:
I think this can be resolved now